src/components/Home/styles.css

```diff
@@ -0,0 +1,33 @@
+.Home {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
```

Comment:
@Mloweedgar If we use 'Home' as the class name, is it possible that we defined this the same name somewhere, so it would lead to conflict/override?
Like I define 'Home' as the class name in another component?

Reply:
Yes, there is a possibility @dadiorchen , My Assumption is that names for our react components are unique (eg. the will be only one Home component), I therefore use the component name as a namespace for my css classes such that all css classes for styling home component will be preceded by 'Home' to make sure those styles only apply to Home component.

Reply:
My idea for improving this could be us decide to use a tool that will ensure every css class we define is made unique, we could for example decide to use tool like [CSS Modules](https://github.com/css-modules/css-modules) or any other similar tool

I will also be happy to hear your advice on this @dadiorchen

Reply:
I don't have any idea for the time being, but I think sperate the code and style is good for a clear and re-useable codebase.

Reply:
Good job @Mloweedgar

Reply:
Thanks @dadiorchen!